AllReadyApp/Web-App/AllReady/Features/Notifications/NotifyAdminForUserUnenrolls.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using AllReady.Areas.Admin.Features.Activities;
+using AllReady.Features.Login;
+using AllReady.Models;
+using MediatR;
+using Microsoft.Extensions.OptionsModel;
+
+namespace AllReady.Features.Notifications
+{
+    public class NotifyAdminForUserUnenrolls : INotificationHandler<UserUnenrolls>
+    {
+        private readonly IMediator _bus;
+        private readonly IOptions<GeneralSettings> _options;
+
+        public NotifyAdminForUserUnenrolls(IMediator bus, IOptions<GeneralSettings> options)
+        {
+            _bus = bus;
+            _options = options;
+        }
+
+        public void Handle(UserUnenrolls notification)
+        {
+            var model = _bus.Send(new UserUnenrollsNotificationQuery {ActivityId = notification.ActivityId});
+            var campaignContact = model.CampaignContacts.SingleOrDefault(tc => tc.ContactType == (int)ContactTypes.Primary);
+
+            if (string.IsNullOrWhiteSpace(campaignContact?.Contact.Email)) return;
+
+            var signup = model.UsersSignedUp.FirstOrDefault(s => s.User.Id == notification.UserId);
+
+            if (signup == null) return;
+
+            var activityLink = $"View activity: {_options.Value.SiteBaseUrl}Admin/Activity/Details/{model.ActivityId}";
+            var subject = $"A volunteer has unenrolled from {model.ActivityName}";
+
+            var message = new StringBuilder();
```

Comment:
This will work for now. @BillWagner and @tonysurma we should talk about getting the templates idea on sendgrid going on the standup this weekend.